wdk-ddi-src/content/fwpsk/ns-fwpsk-fwps_callout2_.md

```diff
@@ -184,6 +184,17 @@ A callout driver specifies this flag when registering a callout that will be add
 </td>
 </tr>
 
+<tr>
+<td width="40%"><a id="FWP_CALLOUT_FLAG_ALLOW_USO"></a><a id="fwp_callout_flag_allow_uso"></a><dl>
+<dt><b>FWP_CALLOUT_FLAG_ALLOW_USO</b></dt>
+<dt>0x00000100</dt>
+</dl>
+</td>
+<td width="60%">
+A callout driver specifies this flag to indicate that the callout supports UDP segmentation offload (USO) with packets larger than the MTU of the network medium. If this flag is not specified, and a callout is registered, then USO is disabled for all traffic that is processed by any filters that specify the callout for the filter's action. <b>Note:</b> The earliest version on which this flag is supported is Windows Insider Preview, version 10.0.25876.

```

Comment:
Are there any restrictions we want to document here for a callout specifying the USO flag?

For example, the URO flag mentions:
<b>Note:</b> If this flag is specified then callouts must not clone and reinject inbound URO packets.

Reply:
All those are supported for USO packets. Only restriction we have currently is w.r.t to the builds this feature is available on which has been mentioned.